packages/language-server/src/plugins/typescript/languageService.ts

```diff
@@ -174,22 +163,19 @@ async function createLanguageService(tsconfigPath: string, workspaceRoot: string
   }
 }
 
-/**
- * This should only be used when there's no jsconfig/tsconfig at all
- */
 function getDefaultJsConfig(): {
-  compilerOptions: ts.CompilerOptions;
-  include: string[];
+   compilerOptions: ts.CompilerOptions;
```

Comment:
Nit: **3-space has entered the chat** 😂

Reply:
Do we not have formatting set up on CI? I guess we didn't do that here yet.

Reply:
I ran it locally and there are a *lot* of changes. Probably should submit that as a different PR and set it up with CI like we have in the main repo.